### PR TITLE
created load_depth_charts

### PR DESCRIPTION
**Pull request by john-b-edwards**
* created function to load depth chart data from `nflfastr-roster`, `load_depth_charts()`
* documented `load_depth_charts()`
* added tests for `load_depth_charts()`

### CODE REVIEW

**Comment by tanho63** [comment]
Updated the naming of dc_ to depth_charts_ in the repo that will need updated I think

File: R/load_nflverse.R [hunk 1]
```diff
@@ -237,3 +237,36 @@ load_teams <- function(){
   class(out) <- c("tbl_df","tbl","data.frame")
   out
 }
+
+#' Load Weekly Depth Charts
+#'
+#' @description Loads depth charts for each NFL team for each week back to 2001.
+#'
+#' @param seasons a numeric vector specifying what seasons to return, if `TRUE` returns all available data
+#' @param file_type One of `"rds"` or `"qs"`. Can also be set globally with options(nflreadr.prefer)
```
Comment (tanho63) [inline R/load_nflverse.R:246]: I think you just need to delete this line and redocument

File: R/load_nflverse.R [hunk 2]
```diff
@@ -237,3 +237,36 @@ load_teams <- function(){
...
+#' @export
+load_depth_charts <- function(seasons = most_recent_season()){
+  if(isTRUE(seasons)) seasons <- 2001:most_recent_season()
+  stopifnot(is.numeric(seasons),
+            seasons >= 2001,
+            seasons <= most_recent_season())
+  p <- NULL
+  if (is_installed("progressr")) p <- progressr::progressor(along = seasons)
+  urls <- paste0("https://github.com/nflverse/nflfastR-roster/",
+                 "raw/master/data/seasons/dc_",
```
Comment (tanho63) [inline R/load_nflverse.R:266]: update path to depth_charts_

**Comment by tanho63** [comment]
Just need to re-document now :)

**Comment by john-b-edwards** [comment]
lol i knew i was forgetting something

**Comment by tanho63** [comment]
leaving for Seb to skim tomorrow

**Comment by tanho63** [comment]
@mrcaseb i'm good with this one too, you can do the honours this time

**Pull request merged**